### ISSUE TITLE
Init Script

### ISSUE THREAD

**Issue by jfernandz** [ProfessorKaos64/RetroRig#157]
Hi all ... :D i write here because i'm having many issues with xboxdrv and Wine VM ... I've already built script and service... but initializes it as /dev/input/event18 no as /dev/input/js0 ... i need that it initializes /dev/input/js0 since is the device that can i remap ... (i can choose these 2 interfaces in Wine but... only can remap js0 with jstest-gtk ...) there exist way to start daemon in js0 ?

Or failing ... there exist way to remap event18 with jstest-gtk ? ... 

Thanks so much.

**Comment by beaumanvienna** [comment]
I would leave it as it used to be. For users with a pretty decent internet connectivity it's fine anyway. All the others just have to be a bit more patient... (Those from "the interview" for example :-DDD )

**Comment by ProfessorKaos64** [comment]
haha. Ok so it's back in the menu under Settings > System Upgrades > "Upgrade 14.04 LTS Kernel to 3.14.0". Are you saying we should upgrade the kernel automatically for 14.04 users? That would essentially calling rrs_ubuntu_kernel_upgrade() after distro/release detection or syscall sourcing.

**Comment by beaumanvienna** [comment]
Yes. If a user is experiencing problems with a controller or the like, manually upgrading the kernel is like guessing. This is not in accordance with the RetroRig philosophy.

**Comment by WyRe** [comment]
Now I'm having some issues with service restart... It does not resume properly after hibernate or suspend ... (I am referring to 'sudo service xboxdrv restart' doesn't works, my gamepad green light is flickering and It doesn't keep first player light). I'm running on the same kernel 3.16.0-4-amd64 ...

**Comment by ProfessorKaos64** [comment]
I know JC encountered this issue as well. I never suspend or hibernate, so it went unnoticed for me until now. 

On January 5, 2015 2:41:56 PM EST, Javi <notifications@github.com> wrote:
>Now I'm having some issues with service restart... It does not resume
>properly after hibernate or suspend ... (I am referring to 'sudo
>service xboxdrv restart' doesn't works, my gamepad green light is
>flickering and It doesn't keep first player light). I'm running on the
>same kernel 3.16.0-4-amd64 ... 
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ProfessorKaos64/RetroRig/issues/157#issuecomment-68763876

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by WyRe** [comment]
Not problem :P 

I hope a fix xD and thanks for ur attention :D

**Comment by ProfessorKaos64** [comment]
> Btw, i'm using this script to resume service when hibernate or suspend with pm-utils ... but not works with systemctl. 

Well currently we don't support any release that is on systemd, but of course that is inevitable with Ubuntu.

**Comment by WyRe** [comment]
But... Why don't you support systemd releases? Systemd is a vital process/daemon to start system (at least in Debian, ain't it?)

**Comment by ProfessorKaos64** [comment]
Oh we will. Its that we are currently focused on getting 14.10 working. Arch linux is probably the first one I will try to get working. Maybe this weekend using the Emulation Station version I have up.

**Comment by WyRe** [comment]
Oh, ... great! :P 

Awaiting news!
